resources/language/resource.language.es_es/strings.po

```diff
@@ -650,7 +653,7 @@ msgstr "Vista para Mi lista"
 
 msgctxt "#30166"
 msgid "Main menu items"
-msgstr "Eltos. del menú principal"
+msgstr "Elementos de menú"
 
```

Comment:
DON'T AGREE. No solution is perfect in this case, My solution is using abbreviations and @roliverosc is omitting information by not using the 'main' word. The size of the control is small in this case and there is little room. I think it's more important not to remove information, so respect the previous solution.